src/widgets/geo-search/GeoSearchRenderer.js

```diff
@@ -32,6 +32,41 @@ const refineWithMap = ({ refine, paddingBoundingBox, mapInstance }) => {
   });
 };
 
+const getBoundsWithPadding = ({
+  paddingBoundingBox,
+  bounds,
+  mapInstance,
+  googleReference,
+}) => {
+  const scale = Math.pow(2, mapInstance.getZoom()) || 1;
+
+  const { northEast, southWest } = bounds;
+
+  const toPoint = latLng =>
+    mapInstance
+      .getProjection()
+      .fromLatLngToPoint(
+        new googleReference.maps.LatLng(latLng.lat, latLng.lng)
+      );
+
+  const northEastPoint = toPoint(northEast);
+  const southWestPoint = toPoint(southWest);
+
+  northEastPoint.x = northEastPoint.x + paddingBoundingBox.right / scale;
+  northEastPoint.y = northEastPoint.y - paddingBoundingBox.top / scale;
+
+  southWestPoint.x = southWestPoint.x - paddingBoundingBox.left / scale;
+  southWestPoint.y = southWestPoint.y + paddingBoundingBox.bottom / scale;
+
+  const ne = mapInstance.getProjection().fromPointToLatLng(northEastPoint);
+  const sw = mapInstance.getProjection().fromPointToLatLng(southWestPoint);
+
+  return {
+    northEast: { lat: ne.lat(), lng: ne.lng() },
+    southWest: { lat: sw.lat(), lng: sw.lng() },
+  };
+};
+
```

Comment:
What do you think about trying to merge the two functions that handle bounds?

Reply:
The functions works in two different directions, so I'd rather have two functions. I can refactor them though.

Reply:
Agree it's better to have two separate functions (sorry my comment was not very clear). But since we have almost the same logic in the function we can refactor it a bit I think.

Reply:
agreed